apps/dapp-console-api/migrations/0001_broad_drax.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE IF NOT EXISTS "addresses" (
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"updated_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"id" uuid PRIMARY KEY DEFAULT gen_random_uuid() NOT NULL,
+	"entity_id" uuid,
+	"address" varchar NOT NULL,
+	"verifications" jsonb DEFAULT '{}'::jsonb NOT NULL,
+	"state" varchar DEFAULT 'active' NOT NULL
+);
+--> statement-breakpoint
+CREATE UNIQUE INDEX IF NOT EXISTS "addresses_entity_id_address_index" ON "addresses" ("entity_id","address");--> statement-breakpoint
```

Comment:
this is what i mean in my other command you can see how it's namespaced compared to below.